City-of-Helsinki/berth-reservations-admin
# VEN-931 | Invoicing feature improvements

## Description :sparkles:

* Show orderNumber in InvoiceModal
* Show harbor name in InvoiceModal
* Show unmarked winter storage invoices and their information
* Invoicing component appearance fixes
* Unified terminology, `bill` -> `invoice`
* Unified invoicing translation keys

## Issues :bug:

### Closes :no_good_woman:

* [VEN-931](https://helsinkisolutionoffice.atlassian.net/browse/VEN-931): FE: Add orderID and harbour name to all invoice modals

## Testing :alembic:

### Automated tests :gear:️

* Updated snapshots and mock data

### Manual testing :construction_worker_man:

* Order number and harbor name should be visible in invoice modal
  * This feature is available in customer view
  * Click the invoices in either open invoices or invoice history section
* Leases for unmarked winter storage should now be visible in customer view
* Open invoices should now display as separate card sections
* The status label for invoice history should now display correctly (not stretched to cell)

## Screenshots :camera_flash:

<img width="1464" alt="image" src="https://user-images.githubusercontent.com/15479131/97170792-f3558980-1794-11eb-97a3-a0a93dbb59ae.png">
<img width="418" alt="image" src="https://user-images.githubusercontent.com/15479131/97170821-ffd9e200-1794-11eb-8043-c002d14a199a.png">
